src/features/Instructors/InstructorsPage/index.jsx

```diff
@@ -16,6 +17,7 @@ const InstructorsPage = () => {
   const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
   const dispatch = useDispatch();
   const [currentPage, setCurrentPage] = useState(initialPage);
+  const [isOpen, open, close] = useToggle(false);
```

Comment:
This names does not explain itself what's supposed to should do, maybe:

```suggestion
  const [isOpen, openMolda, closeModal] = useToggle(false);
```

Reply:
done